x-pack/plugins/cases/server/client/cases/push.ts

```diff
@@ -139,32 +141,17 @@ export const push = async (
     }
 
     const alertsInfo = getAlertInfoFromComments(theCase?.comments);
-
     const alerts = await getAlerts(alertsInfo, clientArgs);
-
-    const getMappingsResponse = await casesClientInternal.configuration.getMappings({
```

Comment:
Mapping is hard coded for the moment. I removed unnecessary calls and code. I believe there is no need to maintain and support code that is not used by users.